src/docs/product/dashboards/custom-dashboards/index.mdx

```diff
@@ -77,3 +77,16 @@ on a world map. An example scenario would be 'in which countries are users exper
 A big number visualization displays the current value of a single function. This visualization is well
 suited for high-level aggregates. An example scenario would be 'p95 of all
 transactions'.
+
+
+## Data Set
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+Select between Discover and [Issues](/product/issues/) data through the "Data Set" selector. Choosing "Issues" allows you to sort and display Issues fields and query using Issues filters, such as `is:unresolved` for unresolved issues. This feature is only enabled for "Table" visualizations.
```

Comment:
Wondering if we should call this "events" data rather than "Discover" data since that's how we call it out in the UI. Your call on that. Also, seems like we should mention the querying before the display and sort because the user has to set up the query first. Or am I misunderstanding?

```suggestion
Select between events and [issues](/product/issues/) data with the "Data Set" selector. Choosing "Issues" allows you to query using issues filters, such as `is:unresolved` for unresolved issues and to display and sort by issues fields. This feature only applies to [table](#table-results) visualizations.
```

Reply:
Agreed, on renaming to "events" due to UI. Made the update! You're correct, querying does happen before field display selection and sorting. Updated this!